app/helpers/notifier_helper.rb

```diff
@@ -4,21 +4,23 @@ module NotifierHelper
   include PostsHelper
 
   # @param post [Post] The post object.
-  # @param opts [Hash] Optional hash.  Accepts :html parameter.
+  # @param opts [Hash] Optional hash.  Accepts :length parameters.
   # @return [String] The formatted post.
   def post_message(post, opts={})
-    rendered = opts[:html] ? post.message&.markdownified_for_mail : post.message&.plain_text_without_markdown
-    rendered.presence || post_page_title(post)
+    if post.respond_to? :message
+      post.message.try(:plain_text_without_markdown).presence || post_page_title(post)
+    else
+      I18n.t "notifier.a_post_you_shared"
+    end
   end
 
   # @param comment [Comment] The comment to process.
-  # @param opts [Hash] Optional hash.  Accepts :html parameter.
   # @return [String] The formatted comment.
   def comment_message(comment, opts={})
     if comment.post.public?
-      opts[:html] ? comment.message.markdownified_for_mail : comment.message.plain_text_without_markdown
+      comment.message.plain_text_without_markdown
     else
-      I18n.translate "notifier.a_limited_post_comment"
+      I18n.t "notifier.a_limited_post_comment"
     end
   end
```

Comment:
These helpers also get used for the notification mails (hence the name "NotifierHelper"), you can't just completely change it's functionality and drop html support. But it looks like you aren't even using these helpers anymore, so maybe just revert these changes?

Reply:
Reverted this.